src/point_one/fusion_engine/messages/control.h

```diff
@@ -570,7 +570,23 @@ struct P1_ALIGNAS(4) EventNotificationMessage : public MessagePayload {
 struct P1_ALIGNAS(4) ShutdownRequest : public MessagePayload {
   static constexpr MessageType MESSAGE_TYPE = MessageType::SHUTDOWN_REQUEST;
   static constexpr uint8_t MESSAGE_VERSION = 0;
-  /** A bitmask of flags associated with the event. */
+
+  /**
+   * Stop navigation engine if running and flush calibration to non-volatile
+   * storage.
```

Comment:
The calibration mention seems somewhat specific, no? Would this not also flush navigation engine state and/or partially completed calibration progress if still calibrating?